spec/factories/charge_factory.rb

```diff
@@ -40,5 +40,12 @@
         }
       end
     end
+
+    factory :volume_charge do
+      charge_model { 'volume' }
+      properties do
+        { ranges: [] }
```

Comment:
Can we define a valid factory?

I mean, here it will return `:missing_ranges` each time we're trying to use that factory, right?

Reply:
Good catch